recipes/msfragger/msfragger.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+#
+# Wrapper script for Java Conda packages that ensures that the java runtime
+# is invoked with the right options. Adapted from the bash script (http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in/246128#246128).
+#
+
+import os
+import sys
+import subprocess
+from os import access, getenv, X_OK
+
+jar_file = f'{os.environ["CONDA_PREFIX"]}/tools/MSFragger.jar'
```

Comment:
If you move `msfragger.py` in the same directory as `MSFragger.jar`, then this can become:
```python
jar_file = os.path.join(os.path.dirname(__file__), "MSFragger.jar")
```